### ISSUE TITLE
possible to build upon gtk4?

### ISSUE THREAD

**Issue by enihcam** [Alex313031/thorium#817]
**Is your feature request related to a problem? Please describe.**

too many gui dependencies are running in my Linux box.

**Describe the solution you'd like, including relevant patches or source**

it'd be great if thorium could be compiled with gtk4.

**Did you read the FAQ?**
https://thorium.rocks/faq

**Comment by Alex313031** [comment]
@enihcam This doesn't have enough info for me to know what you mean?

**Comment by enihcam** [comment]
> @enihcam This doesn't have enough info for me to know what you mean?

https://github.com/Alex313031/thorium/blob/bd305b9d6a90987cada850ea5b507417375c4cd7/src/chrome/browser/thorium_flag_entries.h#L164

is this a compile option or runtime option? how to make thorium to use gtk4 backend?

**Comment by gz83** [comment]
This is a runtime flag, you can find it at chrome://flags

**Comment by Alex313031** [comment]
@enihcam Yeah, just go to `chrome://flags/#gtk-version` and you can select it there.
